go/apps/ctrl/services/routing/service_test.go

```diff
@@ -0,0 +1,682 @@
+package routing
```

Comment:
I think we can get delete this test file, it currently just tests mocked up values that are hardcoded/generated so that it passes, and not the actual the service implementation.